CMSIS/NN/NN_Lib_Tests/nn_test/Ref_Implementations/arm_convolve_HWC_q15_ref.c

```diff
@@ -43,7 +43,7 @@ void arm_convolve_HWC_q15_ref(const q15_t * Im_in,  // input image
         {
             for (k = 0; k < dim_im_out; k++)
             {
-#if defined (ARM_NNUSE_ROUND)
+#ifndef ARM_NN_TRUNCATE
```

Comment:
I guess this one, and similar, are intended to use NN_ROUND macro instead of this ifdefs.

Reply:
This is the reference implementation to test the correctness of kernel functions. I kept the old implementation so that I can test whether the NN_ROUND macro is working as expected or not.